ai-ml/trainium-inferentia/variables.tf

```diff
@@ -49,3 +49,45 @@ variable "enable_mpi_operator" {
   type        = bool
   default     = false
 }
+
+variable "inf2_8xl_min_size" {
+  description = "Inf2-xl Worker node minimum size"
+  type = number
+  default = 0
+}
+
+variable "inf2_8xl_desired_size" {
+  description = "Inf2-xl Worker node desired size"
+  type = number
+  default = 0 // adjust for your testing needs
+}
+
+variable "inf2_8xl_max_size" {
+  description = "Inf2-xl Worker node maximum size"
+  type = number
+  default = 2
+}
+
+variable "inf2_capacity_type" {
+  description = "Inf2 Worker node capacity type"
+  type = string
+  default = "ON_DEMAND" // change to "SPOT" after testing
+}
+
+variable "inf2_24xl_min_size" {
+  description = "Inf2-24xl Worker node minimum size"
+  type = number
+  default = 0
+}
+
+variable "inf2_24xl_desired_size" {
```

Comment:
you can rebase from the main as this code is already in the main

Reply:
Sure, will do.